src/commands/marketplace/publish.ts

```diff
@@ -99,7 +100,7 @@ export default class MarketplacePublish extends Command {
   private async upload(buffer: Buffer): Promise<string> {
     const res = await this.IPFS.add(Buffer.from(buffer), {pin: false})
     if (!res.length) {
```

Comment:
let's use the require function here to not have to manage errors, commands should not manage directly errors to simplify it